subsystem.py

```diff
@@ -0,0 +1,94 @@
+robot_path = 'src/main/java/frc/robot/'
+subsystem_class = '''package frc.robot.subsystems;
+
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+
+import frc.robot.maps.subsystems.${CLASSNAME}Map;
+
+public class ${CLASSNAME} extends SmartSubsystemBase {
+
+    private ${CLASSNAME}Map map;
+
+    public ${CLASSNAME}(${CLASSNAME}Map map) {
+        this.map = map;
+    } 
+
+    @Override
+    public void reset() {
+        // Nothing to reset here
+    }
+
+    @Override
+    public void periodic() {
+        // This method will be called once per scheduler run
+        // Use this for any background processing
+    }
+
+    @Override
+    public void simulationPeriodic() {
+        // This method will be called once per scheduler run during simulation
+    }
+
+    @Override
+    public void safeState() {
+
+    }
+}'''
+
+subsystem_map = '''package frc.robot.maps.subsystems;
+
+public class ${CLASSNAME}Map {
+    public ${CLASSNAME}Map() {
+
+    }
+}
+'''
+
+def insert_into(filename: str, keyword: str, line:str, before:bool = False):
+    with open(f'{robot_path}{filename}','r') as fp:
+        contents = fp.read()
+    if before:
```

Comment:
Pro tip - rather than duplicating the whole replace line, assign just the replacement line. Then you only have the function call in one place.